packages/components/src/Actions/Actions.component.js

```diff
@@ -108,15 +86,12 @@ function Actions(props) {
 Actions.propTypes = {
 	actions: PropTypes.arrayOf(
 		PropTypes.oneOfType([
-			PropTypes.shape(Action.propTypes),
-			PropTypes.shape({
-				displayMode: PropTypes.oneOf([TYPE_DROPDOWN, TYPE_SPLIT_DROPDOWN]),
-			}),
+			PropTypes.shape(ActionButton.propTypes),
 		]),
 	),
 	className: PropTypes.string,
 	hideLabel: PropTypes.bool,
-	tooltipPlacement: Action.propTypes.tooltipPlacement,
+	tooltipPlacement: ActionButton.propTypes.tooltipPlacement,
```

Comment:
it is weird  to render abstract Action and use ActionButton as PropType provider

Reply:
exact